### PR TITLE
Fix loss calculation in tms.ipynb

### PR DESCRIPTION
**Pull request by sonofhypnos**
Fixed incorrect calculation of the loss for the log. Multiplied loss by batch size to accumulate sum instead of mean loss before dividing by total length. The previous code didn't work as torch.nn.MSELoss returns [the mean of batch samples by default.](https://pytorch.org/docs/stable/generated/torch.nn.MSELoss.html)

### CODE REVIEW

**Comment by sonofhypnos** [comment]
Yes, it doesn't affect training, only the logs. It first confused me when I was re-using the code from the notebook and comparing runs with different batch sizes.
